CDP4Grapher/Views/Grapher.xaml

```diff
@@ -56,6 +56,14 @@
                                 </behaviors:GrapherOrgChartBehavior>
                             </dxmvvm:Interaction.Behaviors>
                         </views1:GrapherDiagramControl>
+
+                        <TextBlock Grid.Row="2"
+                                   TextAlignment="Center"
+                                   VerticalAlignment="Center"
+                                   HorizontalAlignment="Center"
+                                   Visibility="{Binding GrapherPlaceholderVisibility}">
+                            The current Iteration for this option does not contain a Top Element
```

Comment:
let s rephrase to:

"The Graph cannot be rendered as the Iteration does not contain a Top Element"